Makefile

```diff
@@ -191,7 +191,8 @@ test-containerized: run-etcd $(BUILD_CONTAINER_MARKER)
 	docker run -ti --rm --privileged --net=host \
 	-e PLUGIN=calico \
 	-v ${PWD}:/go/src/github.com/projectcalico/calico-containers:rw \
-	$(BUILD_CONTAINER_NAME) make ut
+	$(BUILD_CONTAINER_NAME) make ut; \
```

Comment:
This line should be `$(BUILD_CONTAINER_NAME) bash -c 'make ut; \` 
That should fix the CircleCI build